docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,179 @@
+```code
+$ tctl auth export --type=windows > teleport.cer
+```
 
-![Install Active Directory](../../img/desktop-access/install-active-directory.png)
+Copy this certificate to your Windows system, if you didn't run `tctl` from there.
 
-Once the server is restarted from one or both command executions, click **NEXT**.
+### Install the Teleport service for Windows
 
-Copy and paste the provided command into a Windows PowerShell window to download
-and run the 'configure Active Directory' script:
+From the Windows system, download the [Teleport service installer](#LinkToCome) {/*Rename as needed */}
+and run it. When prompted, select the Teleport certificate file from the previous step.
+Once complete, reboot the system.
 
-![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
+## Step 2/4. Install the Teleport Desktop Service
 
-Click **NEXT**.
+On the Linux host where you will run the Desktop Service, install Teleport (if
+you plan on running the Windows Service on the same host as an existing Teleport
+service, you can skip this step):
 
-The PowerShell script will output a Teleport configuration block. Copy this
-block to a temporary location. Click **Next**.
+(!docs/pages/includes/install-linux.mdx!)
 
-On the Linux host you installed Teleport to run as the Desktop Access connector,
-edit `/etc/teleport.yaml` and paste the configuration provided by the output of
-the previous step. Review and edit the addition as necessarily for your environment:
+Enable the Desktop Service and list the Windows systems configured for
+access through Teleport in `/etc/teleport.yaml`.
 
-```yaml
+```yml
 version: v3
 teleport:
-  auth_token: 2239...c5b21
-  proxy_server: teleport.example.com:443
+  nodename: example.teleport.com
+  proxy_server: teleport-proxy.example.com:443
+windows_desktop_service:
+  enabled: yes
+  non_ad_hosts:
+    - 192.0.2.156
+```
 
-auth_service:
-  enabled: no
-ssh_service:
-  enabled: no
-proxy_service:
-  enabled: no
+<Details title="Add labels to hosts">
+You can attach labels to your Windows hosts by matching to their hostnames.
+For example, to add the `cloud: ec2` label to hosts with EC2 private IP DNS names:
 
+```diff
+version: v3
+teleport:
+  nodename: example.teleport.com
+  proxy_server: teleport-proxy.example.com:443
 windows_desktop_service:
   enabled: yes
-  ldap:
-    addr:     '10.10.1.50:636'
-    domain:   'windows.teleport.example.com'
-    username: 'WIN\svc-teleport'
-    server_name: 'windows-server-hostname'
-    insecure_skip_verify: false
-    ldap_ca_cert: |
-        -----BEGIN CERTIFICATE-----
-        MIIDnzCCAoegAwIBAgIQT/UIn+MT4aZC9ix/QuiV9zANBgkqhkiG9w0BAQsFADBi
-        ...
-        31qA4dO3if7RdikD9hVbiIF9jQ==
-        -----END CERTIFICATE-----
-
-  discovery:
-    base_dn: '*'
-  labels:
-    teleport.internal/resource-id: 42d8859c-60d0-4d7f-9767-bdd66b63fce6
+  non_ad_hosts:
+-    - 192.0.2.156
++    - ip-192-0-2-156.us-east-2.compute.internal
++ host_labels:
```

Comment:
@probakowski can you confirm that `host_labels` apply to `non_ad_hosts` too? (they should, but I'm not sure if we implemented it)

Reply:
I'm 90% sure I confirmed this through testing.

Reply:
Just confirmed that it does apply